parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -470,7 +474,11 @@ installCNI() {
         echo "WARNING: no cni-plugins components present falling back to hard coded download of 1.4.1. This should error eventually" 
         # could we fail if not Ubuntu2204Gen2ContainerdPrivateKubePkg vhd? Are there others?
         # definitely not handling arm here.
```

Comment:
Can we add a log_to_event with a pattern for tracing the URL used to download the package

Reply:
not sure I understand the point here, logs_to_events should really only be used to gather latency telemetry for individual CSE steps, not tracing variable values